### ISSUE TITLE
docs/user-guide/walkthrough/README.md : Fix the get pod command

### ISSUE THREAD

**Issue by dpkshetty** [kubernetes/kubernetes#14346]
Wrt section 
https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/walkthrough/README.md#pod-management
```
$ curl http://$(kubectl get pod nginx -o go-template={{.status.podIP}})
```
is incorrect

It should rather be

```
$ curl http://$(kubectl get pod nginx -t {{.status.podIP}})
```

I can send a doc patch if the issue reported above is correct ?

**Comment by gmarek** [comment]
cc @bgrant0607

**Comment by erictune** [comment]
You are looking at docs for HEAD.  They are correct for code built at HEAD.

If you are using a released binary, v1.0.x, then you should read the docs here:
http://kubernetes.io/v1.0/docs/user-guide/walkthrough/README.html
which say:
`curl http://$(kubectl get pod nginx -o=template -t={{.status.podIP}})`, which should work with a v1.0 kubectl.
